### ISSUE TITLE
I reinstalled Chrome and lost my rules, didn't back them up anywhere... now rewriting and having issue with end page

### ISSUE THREAD

**Issue by Auditormadness9** [hoothin/UserScripts#530]
Everytime I it splices next page, it says "reached end page" and doesn't go further than Page 2 unless manually clicked on Page 3, then it would reach Page 4 and do same again. Any way to increase the time it waits until it decides if it's the last page or not?

**Comment by hoothin** [comment]
Which site and what rule is using?

**Comment by Auditormadness9** [comment]
Was there no way to set the max delay? I just dunno if I feel comfo sharing the site.

**Comment by hoothin** [comment]
No, there is no `delay`, I think the next page of this website is dynamically loaded, which limits the ability to load more than one page. However, without providing the specific website URL, I cannot assess the situation and offer assistance.

**Comment by Auditormadness9** [comment]
Sure, here's an example test page: https://motherless.com/term/videos/test

**Comment by hoothin** [comment]
I've updated the build-in rule for that site.
